src/pages/my-ads/screens/MyAds/MyAdsTableRow/MyAdsTableRow.scss

```diff
@@ -26,7 +26,7 @@
         position: relative;
         display: grid;
         align-items: center;
-        grid-template-columns: repeat(2, 1.6fr) 1.8fr 1.9fr 3fr 1.9fr;
+        grid-template-columns: 1.55fr 1.55fr 1.7fr 1.9fr 3fr 1.9fr;
```

Comment:
changed the sizes a bit because the payment methods was overlapping with the Active and Inactive status in My Ads page